src/translator/service.h

```diff
@@ -134,6 +135,13 @@ class Service {
   /// Shortlist memory passed as bytes.
   AlignedMemory shortlistMemory_;  // ORDER DEPENDENCY (translators_)
 
+  QualityEstimator qualityEstimator_;
+
+  /// Holds instances of batch translators, just one in case
+  /// of single-threaded application, numWorkers_ in case of multithreaded
+  /// setting.
+  std::vector<BatchTranslator> translators_;  // ORDER DEPENDENCY (modelMemory_, shortlistMemory_)
```

Comment:
What's this?  Looks like a merge failure.

Reply:
You're right. Just remove it 97cfe06